src/exchanges/bitmex/const.js

```diff
@@ -26,7 +26,7 @@ export const API_RESOLUTIONS_MAP_UDF = {
 };
 
 export const API_OPTIONS = {
-  apiLimit: 750,
+  apiLimit: 3000,
```

Comment:
where exactly is this const used?

Reply:
Here:

https://github.com/wnz99/tvcd/blob/master/src/utils/makeChunkCalls.js#L18

Some exchanges have a maximum number of candles returned in a singled call, so it could happens that the total number of candles from time `x` to time `y` is 10000, but the API will return only a maximum of 1500 for each single API call. This would cause gaps in the chart being displayed by TV.

To overcome this issue, we split the data fetching in `n` calls for `1500` candles each. This behaviors is controlled by that constant. Obviously, an higher number of calls could trigger rate limiting in the exchange API, if present.

Reply:
i see, from reading your code it sounds to me the variable name could be "numberOfCandles" instead of apiLimit ?

apiLimit sounds like you're counting the rate limit or number of requests?

how does they rate limit the candles api, is it by the bandwidth we use per request or by the number of requests per minute ? do they return the rate limit status every time you do requests or do we have to calculate ourselves?